internal/envtest/envtest.go

```diff
@@ -101,13 +118,45 @@ func (e *Environment) stop() {
 
 func (e *Environment) CreateNamespace(t *testing.T) string {
 	namespace := strings.ToLower(t.Name())
-	if err := kubernetes.CreateNamespaceIfNotExists(e.intClient, namespace); err != nil {
+	if err := kubernetes.CreateNamespaceIfNotExists(e.Client, namespace); err != nil {
 		t.FailNow()
 	}
 
 	return namespace
 }
 
-func (e *Environment) RestConfig() *rest.Config {
-	return e.env.Config
+func (e *Environment) GetKubeconfig(t *testing.T) ([]byte, error) {
```

Comment:
issue: There's unnecessary t *testing.T here. Lets keep that in the test code itself.

Reply:
Nice catch, thank you.